src/chi-vue/src/constants/classes.ts

```diff
@@ -141,6 +160,16 @@ export const GENERIC_SIZE_CLASSES = {
 
 //#region Icon
 export const ICON_CLASS = 'chi-icon';
+
+export const ICON_CLASSES = {
```

Comment:
I like the idea of having a list of icons like this but these should not be here.
There are too many icons to list them all in `classes.ts`.
Maybe we could create `icons.ts` and define them there.